frontend/src/RecipeCard.tsx

```diff
@@ -9,9 +9,11 @@ export default function RecipeCard(props: RecipeProps) {
 
     return (
         <div className="recipe-card">
-            <p>name:</p>
+            <p>Name:</p>
             {props.recipe.name}
-            <p>id:</p>
+            <p>Description:</p>
+            {props.recipe.description}
+            <p>ID:</p>
             {props.recipe.id}
```

Comment:
Ich würde jeglichen Text in ein <p> Tag packen. Hier könnte man dann überlegen auch nur ein p-Tag mit <br> zu nutzen. Oder vielleicht den Namen vom Rezept in ein h1/2/... Tag?

Reply:
Umschließenden <p> Tag definiert.